modules/backend/formwidgets/repeater/assets/js/repeater.js

```diff
@@ -134,6 +137,16 @@
         return defaultText
     }
 
+    Repeater.prototype.hidePrompt = function () {
+        var maxItems = $('.field-repeater').attr('data-max-items')
```

Comment:
All direct references to jQuery selectors within this method need to be changed, they are too broad. Try testing with multiple repeaters on the same form and you'll see what problems these broad selectors will cause.

Try using `this.$el` to be more specific.

Also, please restructure to store the maxItems data in the widget JS options property. See https://github.com/octobercms/october/blob/master/modules/backend/formwidgets/repeater/assets/js/repeater.js#L33-L37 and https://github.com/octobercms/october/blob/master/modules/backend/formwidgets/repeater/assets/js/repeater.js#L147 for reference. Look into what `$.data('oc.repeater')` returns and see how you can best integrate passing the option from the FormWidget options in PHP to the JS that runs this.

Thank you for your work on this so far, keep it up!